### ISSUE TITLE
build revision is ignored

### ISSUE THREAD

**Issue by bparees** [fabric8io/openshift-jenkins-sync-plugin#108]
builds have both a ref(build.spec.source.git.ref) and a revision(build.spec.revision) associated with them.

revision is supplied when the user wants to build a specific commit when invoking "oc start-build --commit=foo"

it looks like the sync plugin just looks at spec.source, but spec.revision should take precedence if present.

(yes it's a bad api and i apologize in advance)

**Comment by tlalexan** [comment]
I believe I fixed this with https://github.com/jenkinsci/openshift-sync-plugin/pull/36 .  Should I submit a pull request into this repository instead?  I'm unclear on the relationship between the openshift-sync-plugin repositories.

**Comment by bparees** [comment]
yeah we've been doing some restructuring.  this is now the upstream repo, so if you can submit the PR here that'd be great.

**Comment by tlalexan** [comment]
Okay, I made a pull request here.  Let me know... I've tried it out, and this seems to be the way to pass a git revision into a Jenkins build
